junos/resource_system_test.go

```diff
@@ -90,6 +90,10 @@ func TestAccJunosSystem_basic(t *testing.T) {
 							"no_redirects", "true"),
 						resource.TestCheckResourceAttr("junos_system.testacc_system",
 							"no_redirects_ipv6", "true"),
+						resource.TestCheckResourceAttr("junos_system.testacc_system",
+							"no_multicast_echo", "true"),
+						resource.TestCheckResourceAttr("junos_system.testacc_system",
+							"default_address_selection", "true"),
```

Comment:
testacc fails : the 2 arguments is missing in return of testAccJunosSystemConfigCreate() function